Model/ConfigProvider.php

```diff
@@ -240,7 +241,7 @@ public function checkConn()
     }
 
     /**
-     * @return \Magento\Payment\Model\MethodInterface
+     * @return mixed
      */
     protected function getData()
     {
```

Comment:
`ccTokenPrimaryOption()` is missing the type for the `$collection` parameter.

Reply:
Done